README.md

```diff
@@ -1,3 +1,284 @@
+
+What do we need to do in order to successfully update a record? We need to first find the appropriate record. Then, we make some changes to it and finally, save it once again. 
+
+In our Ruby ORM, in which attributes of a given Ruby objects are stored as an individual row in a database table, we will need to retrieve these attributes, reconstitute them into a Ruby object, make changes to that object using Ruby methods and *then* save those (newly updated) attributes back into the database. 
+
+Let's walk through this process together. 
+
+## Updating Record in a Ruby ORM
+
+For the purposes of this example, we'll be working with a fictitious music management app that allows the user to store their songs. Our app has a `Song` class that maps to a songs database table. Our `Song` class has all the methods it needs to create the songs table, insert records into that table and retrieve records from that table. 
+
+### The `Song` Class
+
+For this example, we'll assume that our database connection is stored in the `DB[:conn]` constant. 
+
+```ruby
+class Song
+
+attr_accessor :name, :album
+attr_reader :id
+  
+  def initialize(id=nil, name, album)
+    @id = id
+    @name = name
+    @album = album
+  end
+
+  def save
+    sql = <<-SQL
+      INSERT INTO songs (name, album) 
+      VALUES (?, ?)
+    SQL
+
+    DB[:conn].execute(sql, self.name, self.album)
+    
+  end
+
+  def self.create(name:, album:)
+    student = Student.new(name, album)
+    student.save
+  end
+  
+  def self.find_by_name(name)
+    sql = "SELECT * FROM songs WHERE name = ?"
+    result = DB[:conn].execute(sql, name)[0]
+    Song.new(result[0], result[1], result[2])
+  end
+end
+```
+
+With the `Song` class as defined above, we can create new `Song` instances, save them to the database and retrieve them from the database:
+
+```ruby
+ninety_nine_problems = Song.create(name: "99 Problems", album: "The Blueprint")
+
+Song.find_by_name("99 Problems")
+# => #<Song:0x007f94f2c28ee8 @id=1, @name="99 Problems", @album="The Blueprint">
+```
+
+Now that we see how to create a `Song` instance, save its attributes to the database, retrieve those attributes and use them to re-create a `Song` instance, let's move on to updating records and objects. 
+
+### Updating Songs
+
+In order to update a record, we must first find it:
+
+```ruby
+ninety_nine_problems = Song.find_by_name("99 Problems")
+
+ninety_nine_problesm.album
+# => "The Blueprint"
+```
+
+Uh-oh, 99 Problems is off The Black Album, as we all know. Let's fix this. 
+
+```ruby
+ninety_nine_problems.album = "The Black Album"
+
+ninety_nine_problems.album
+# => "The Black Album"
+```
+
+Much better. Now we need to save this record back into the database. To do so, we'll need to use an `UPDATE` SQL statement. That statement would look something like this:
```

Comment:
So you wouldn't want to run a SQL statement with the `WHERE` clause against the name. There may be *many* songs with the name "99 problems". The only unique identifier for that row is the ID. This example needs to be against an id